sprinklers/base.py

```diff
@@ -42,18 +42,37 @@ class SprinklerBase(object):
     subtask_queue = current_app.conf.CELERY_DEFAULT_QUEUE
     klass = None
 
+    subtask_options = {  # any custom task properties we may want to set, like so:
+        'rate_limit': None
+    }
+    """
+    For a full list of options that can be assigned to subtasks see:
+    :link https://docs.celeryproject.org/en/latest/userguide/tasks.html#list-of-options:
+    """
+
     def __init__(self, **kwargs):
         self.kwargs = kwargs
         if self.klass is None:
             self.klass = self.get_queryset().model
 
+    def async_subtask_options(self):
```

Comment:
```suggestion
    def get_async_subtask_options(self):
```

But I don't feel strongly about it